packages/admin/admin/src/error/errorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,85 @@
+import { Error } from "@comet/admin-icons";
+import { type ComponentsOverrides, type Divider, type Theme, type Typography, useThemeProps } from "@mui/material";
+import { type FunctionComponent, type ReactNode } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Button } from "../../common/buttons/Button";
+import { CometLogo } from "../../common/CometLogo";
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+import { InlineAlert } from "../../inlineAlert/InlineAlert";
+import { ActionContainer, ContentContainer, DividerStyled, Info, LogoContainer, Root, Title } from "./ErrorPage.styles";
+
+export type ErrorPageClassKey = "root" | "title" | "info" | "contentContainer" | "divider" | "logoContainer" | "actionContainer";
+
+export type ErrorPageProps = ThemedComponentBaseProps<{
+    root: "div";
+    contentContainer: "div";
+    iconContainer: "div";
+    divider: typeof Divider;
+    title: typeof Typography;
+    info: typeof Typography;
+    logoContainer: "div";
+    actionContainer: "div";
+}> & {
+    logo?: ReactNode;
+    icon?: ReactNode;
+    title?: ReactNode;
+    description?: ReactNode;
+    help?: ReactNode;
```

Comment:
I don't think it's really clear what should be passed in here, unlike for `title` and `description`. 

```suggestion
    helpText?: ReactNode;
```